crates/re_renderer/src/renderer/depth_cloud.rs

```diff
@@ -245,20 +265,49 @@ impl DepthCloudDrawData {
                 ),
             };
 
-            instances.push(DepthCloudDrawInstance {
-                num_points: depth_cloud.depth_dimensions.x * depth_cloud.depth_dimensions.y,
-                bind_group: ctx.gpu_resources.bind_groups.alloc(
+            let mk_bind_group = |label, draw_data_uniform_buffer| {
+                ctx.gpu_resources.bind_groups.alloc(
                     &ctx.device,
                     &ctx.gpu_resources,
                     &BindGroupDesc {
-                        label: "depth_cloud_bg".into(),
+                        label,
                         entries: smallvec![
-                            ubo,
+                            ubo.clone(),
                             BindGroupEntry::DefaultTextureView(depth_texture.handle),
+                            draw_data_uniform_buffer
                         ],
                         layout: bg_layout,
                     },
+                )
+            };
+
+            let bind_group_outline = mk_bind_group(
+                "depth_cloud_outline_mask".into(),
+                create_and_fill_uniform_buffer(
+                    ctx,
+                    "PointCloudDrawData::DrawDataUniformBuffer_outline_mask".into(),
+                    gpu_data::DrawDataUniformBuffer {
+                        radius_boost_in_ui_points: radius_boost_in_ui_points_for_outlines.into(),
+                        end_padding: Default::default(),
+                    },
+                ),
+            );
+            let bind_group_opaque = mk_bind_group(
+                "depth_cloud_bg".into(),
+                create_and_fill_uniform_buffer(
+                    ctx,
+                    "PointCloudDrawData::DrawDataUniformBuffer".into(),
```

Comment:
I started naming those by their field names and I think that's better practice than what we had before. I.e it would be `DepthCloudDrawInstance::bind_group_opaque` here. Not consistent across everywhere though, jury is still out on this ;)

Reply:
I think you are in a better position to take a rename-pass over all of re_renderer